scripts/zkai_proof_pressure_main_evidence_gate.py

```
@@ -0,0 +1,411 @@
+    parts.append('<text x="48" y="592" fill="#aab2bd" font-family="Inter, Arial, sans-serif" font-size="15">The speed result is different: local proving time grows near the work axis, so the claim must stay about proof boundaries.</text>')
+    parts.append("</svg>")
+    atomic_write(path, "\n".join(parts) + "\n")
+
+
+def validate_output_path(path: pathlib.Path) -> None:
+    if not path.is_absolute():
+        path = ROOT / path
+    try:
+        path.relative_to(EVIDENCE_DIR)
+    except ValueError as err:
+        raise ProofPressureMainEvidenceError("output must stay inside evidence dir") from err
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)